api/swagger.yaml

```diff
@@ -7958,6 +7958,9 @@ paths:
           description: |
             A JSON encoded value of the filters (a `map[string][]string`) to process on the secrets list. Available filters:
 
+            - `id=<secret id>`
+            - `label=<key> or label=<key>=value`
+            - `name=<secret name>`
             - `names=<secret name>`
```

Comment:
@allencloud It looks like docker 17.03 does not have `docker secret ls --filter`. In that case we can remove support for "names", because it has not yet been in a release